configuration/src/main/kotlin/com/malinskiy/marathon/config/vendor/android/VideoConfiguration.kt

```diff
@@ -12,5 +12,6 @@ data class VideoConfiguration(
     @JsonProperty("height") val height: Int = 1280,
     @JsonProperty("bitrateMbps") val bitrateMbps: Int = 1,
     @JsonProperty("timeLimit") val timeLimit: Long = 180,
-    @JsonProperty("timeLimitUnits") val timeLimitUnits: TimeUnit = TimeUnit.SECONDS
+    @JsonProperty("timeLimitUnits") val timeLimitUnits: TimeUnit = TimeUnit.SECONDS,
+    @JsonProperty("longVideoSupport") val longVideoSupport: Boolean = false
```

Comment:
We should not ask for any input here from the user. Same as the video recording falling back to screenshots automatically. Please remove

Reply:
but it's the toggle to enable new behavior - how we can control it wo the toggle?

Reply:
There is no reason to control this. There are only 2 cases here:
- users who disabled video will still have the same behavior
- users who enabled video will get the extended recording if their tests are longer than 3 minutes

I dont see any reason to have this behaviour behind a flag: enhancements that do not break curremt behaviors should not be configurable by users.

If this was a breaking change then yes it would be required, but this is not the case

Reply:
done